packages/contracts-bedrock/src/L2/OptimismSuperchainERC20Factory.sol

```diff
@@ -13,22 +12,22 @@ import { CREATE3 } from "@rari-capital/solmate/src/utils/CREATE3.sol";
 /// @title OptimismSuperchainERC20Factory
 /// @notice OptimismSuperchainERC20Factory is a factory contract that deploys OptimismSuperchainERC20 Beacon Proxies
 ///         using CREATE3.
-contract OptimismSuperchainERC20Factory is IOptimismERC20Factory, ISemver {
-    /// @notice Mapping of the deployed OptimismSuperchainERC20 to the remote token address.
-    ///         This is used to keep track of the token deployments.
-    mapping(address _superchainToken => address remoteToken_) public deployments;
-
+contract OptimismSuperchainERC20Factory is ISemver {
     /// @notice Emitted when an OptimismSuperchainERC20 is deployed.
-    /// @param superchainToken  Address of the SuperchainERC20 deployment.
+    /// @param superchainToken  Address of the OptimismSuperchainERC20 deployment.
     /// @param remoteToken      Address of the corresponding token on the remote chain.
     /// @param deployer         Address of the account that deployed the token.
     event OptimismSuperchainERC20Created(
         address indexed superchainToken, address indexed remoteToken, address deployer
     );
 
     /// @notice Semantic version.
-    /// @custom:semver 1.0.0-beta.3
-    string public constant version = "1.0.0-beta.3";
+    /// @custom:semver 1.0.0-beta.4
+    string public constant version = "1.0.0-beta.4";
+
+    /// @notice Mapping of the deployed OptimismSuperchainERC20 to the remote token address.
+    ///         This is used to keep track of the token deployments.
+    mapping(address _localToken => address remoteToken_) public deployments;
```

Comment:
Naming nits:
1. Is there a benefit for using underscores in named mappings? The name can't be used anywhere AFAIK and is only for readability, so may as well remove the underscores
2. Naming the second arg in a mapping always seemed redundant to me, because it's the same as the variable. In other words, it feels confusing to me that `remoteToken_` and `deployments` are the same thing—doesn't this mean we're giving two variable names to the same value? Using:
```solidity
mapping(address _localToken => address) public deployments;
```
or
```solidity
mapping(address _localToken => address) public remoteTokens;
```
feels more clear to me